packages/esm-ward-app/src/index.ts

```diff
@@ -19,12 +20,17 @@ const options = {
 
 export const root = getSyncLifecycle(rootComponent, options);
 
+export const wardDashboardLink = getSyncLifecycle(createDashboardLink({ name: 'ward', title: 'wards' }), options);
```

Comment:
The dashboard meta file seemed like a needless extraction, I just included the "name" and "title" parameters here.